src/app/tests/suites/certification/Test_TC_TSUIC_1_1.yaml

```diff
@@ -53,7 +53,18 @@ tests:
       response:
           constraints:
               type: list
-              contains: [0, 1, 2, 65528, 65529, 65530, 65531, 65532, 65533]
+              contains: [0, 1, 65528, 65529, 65530, 65531, 65532, 65533]
+
+     - label:
+          "Read the optional attribute(ScheduleProgrammingVisibility) in
+          AttributeList"
+      PICS: TSUIC.S.A0002
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [2]
```

Comment:
This implements one side of the "if and only if" from the test plan.  Also needs a test that the attribute is not there if !TSUIC.S.A0002

Reply:
Agreed, but how can that be achieved with the yaml syntax ?

I could not find an example that does this kind of negative check in other 1.1 tests.
for example:
https://github.com/project-chip/connectedhomeip/blob/master/src/app/tests/suites/certification/Test_TC_OO_1_1.yaml
https://github.com/project-chip/connectedhomeip/blob/master/src/app/tests/suites/certification/Test_TC_LVL_1_1.yaml

Maybe it's easier done in a python based test like: https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_DeviceBasicComposition.py in one big sweep for all clusters on the device ?

Reply:
> Agreed, but how can that be achieved with the yaml syntax ?

```
      PICS: "!TSUIC.S.A0002"
      command: "readAttribute"
      attribute: "AttributeList"
      response:
          constraints:
              type: list
              excludes: [2]
```

Reply:
And there is a bunch of it in the newer 1_1 tests: 
```
% git grep -l excludes | grep yaml | grep 1_1
src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_CMOCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_FLDCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_NDOCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_OZCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_PMHCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_PMICONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_PMKCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_RNCONC_1_1.yaml
src/app/tests/suites/certification/Test_TC_TIMESYNC_1_1.yaml
src/app/tests/suites/certification/Test_TC_TVOCCONC_1_1.yaml
```

Reply:
@bzbarsky-apple Thanks for the reference - updated TC with the exclusion part.
Can you re-check ?